File: cfgs/pythoncfgs/HOW_CAN_WE_SUM_THE_DIGITS_OF_A_GIVEN_NUMBER_IN_SINGLE_STATEMENT.py
Nodes
1;statement;foo_1 = 0;8;
2;conditional;foo_1 != 0;9;
3;statement;return foo_1;12;
4;statement;foo_1 = foo_2 + fun_1(foo_3 % 10);10;
5;statement;foo_1 = fun_1(foo_2 / 10);11;
6;exit;;
Edges
0;2;
1;3;4;
2;6;
3;5;
4;2;
5;

File: cfgs/java_cfgs_foo/HOW_CAN_WE_SUM_THE_DIGITS_OF_A_GIVEN_NUMBER_IN_SINGLE_STATEMENT.java
Nodes
1;special;;;
2;statement;int sum = 0;14;
3;conditional;foo_1 != 0;15;
4;statement;sum = foo_1 + (foo_2 % 10);16;
5;statement;n = foo_1 / 10;17;
6;statement;return foo_1;19;
7;exit;;;
Edges
0;2;
1;3;
2;6;4;
3;5;
4;3;
5;7;
6;
